source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,23 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB by using the `watch()
+<https://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#watch>`_ method. ``watch()`` opens a :manual:`Change Stream</changeStreams/>`, that allows applications to watch for changes to data and react to those changes. After opening a ``Change Stream``, you can process the change events using the `changeStream.next()
+<http://mongodb.github.io/node-mongodb-native/3.5/api/ChangeStream.html#next>`_ method. ``watch()`` takes an optional ``pipeline`` parameter that is an array of :manual:`stages </reference/operator/aggregation-pipeline/#stages>` to filter the change events output. You can close a change stream 
+using the `changeStream.close()
+<https://mongodb.github.io/node-mongodb-native/3.5/api/ChangeStream.html#close>`_ method.
+
+Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set the ``fullDocument`` option to the
+string ``updateLookup`` to include the changes to the document as well as
+the entire altered document.
+
+The ``watch()`` command returns a ``Change Stream`` instance. The ``changeStream.next()`` method returns a promise that resolves to an object with the change events. The ``documentKey`` field of this object is an object that contains the _id of the changed document.
```

Comment:
Nitpick: Let's be consistent with capitalization of the change stream term whether capitalized. This should also apply to the term "change event" and "object".

Reply:
Nitpick: could you use rst preformatted for "_id" just for clarity that it's a specific field?

Reply:
Fixed: 
* Made capitalization consistent for 'Change Stream'
* Formatted _id to ``_id``